gulpfile.js

```diff
@@ -1,47 +1,46 @@
-var gulp = require('gulp');
-var minify = require('gulp-minify');
-var minifyCSS = require('gulp-clean-css');
-var concat = require('gulp-concat');
-var del = require('del');
+const { series, parallel, src, dest, pipe} = require('gulp');
+const minify = require('gulp-minify');
+const minifyCSS = require('gulp-clean-css');
+const concat = require('gulp-concat');
+const del = require('del');
 
-gulp.task('clean_js', function() {
+function cleanJs(cb) {
   return del(['ooyala_player/public/build/*.js']);
-});
+  cb();
+}
 
-gulp.task('clean_css', function() {
+function cleanCSS(cb){
```

Comment:
I think the function name should be cleanCss. like we are using for the upper function.